tests/bwc/test_rolling_upgrade.py

```diff
@@ -225,6 +255,18 @@ def _test_rolling_upgrade(self, path, nodes):
                 # Add the shards of the new partition primaries
                 expected_active_shards += shards
 
+                if int(path.from_version.split('.')[0]) >= 5:
+                    c.execute(f"insert into doc.t2(a, b) values ({idx}, {idx})")
+                    c.execute("refresh table t2")
+                    c.execute(f"select a, b, c>=1 and c<=2, d>a+b from doc.t2 where a = {idx}")
+                    self.assertEqual(c.fetchall(), [[idx, idx, True, True]])
+                    if int(path.from_version.split('.')[1]) >= 4:
```

Comment:
There are a few bugs that are fixed in later releases so it cannot be merged with existing parted table and tested for version older than `5.4`.